bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestAppProfile.java

```diff
@@ -145,17 +145,6 @@ public void testMutateRow() throws Exception {
     Assert.assertEquals(req2.getAppProfileId(), "my-app-profile");
   }
 
-  @Test
-  public void testMutateRows() throws Exception {
```

Comment:
instead of deleting, should we replace the test with the non deprecated method `getDataClientWrapper()`?

Reply:
We should keep both apis. Please undeprecate the old nonwrapped clients  ie getTableAdminClient and please mark all of the wrapper clients as internal api. They should be used solely by the hbase layer